modules/installation-configuration-parameters.adoc

```diff
@@ -1052,6 +1100,30 @@ within link:https://azure.microsoft.com/en-us/global-infrastructure/regions[a re
 |The availability zones where the installation program creates control plane machines.
 |String list
 
+|`platform.azure.defaultMachinePlatform.settings.securityType`
+|Enables confidential VMs or trusted launch for all nodes. This option is not enabled by default.
+|`ConfidentialVM` or `TrustedLaunch`.
+
+|`platform.azure.defaultMachinePlatform.settings.confidentialVM.uefiSettings.secureBoot`
+|Enables secure boot on all nodes if you are using confidential VMs.
+|`Enabled` or `Disabled`. The default is `Disabled`.
+
+|`platform.azure.defaultMachinePlatform.settings.confidentialVM.uefiSettings.virtualizedTrustedPlatformModule`
```

Comment:
I realize this is probably unavoidable at this point, but this is really unfortunate how this table looks, with these wide of parameter names.

Might be worth revisiting this table in a future jira

Reply:
Yeah, that's definitely going to be a future enhancement I want to chase down. The table is hard to use